test/meson.build

```diff
@@ -24,7 +21,8 @@ if gtest_dep.found() and not meson.is_cross_build()
                               link_args: extra_link_args,
                               dependencies : deps + [gtest_dep],
                               build_rpath : '$ORIGIN')
-        test(test_name, test_exe, timeout : 120)
+        test(test_name, test_exe, timeout : 120,
+             workdir: meson.current_build_dir())
```

Comment:
I would put this commit (a06260815) at the beginning of the PR as it is can be seen as a "preparatory" commit.

Reply:
Done